src/modules/todo/components/TodoDetail.js

```diff
@@ -16,6 +17,19 @@ import TodoForm from './TodoForm'
 
 class TodoDetail extends React.Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {loader: false}
```

Comment:
You don't have to use additional loading state.
Just use fetchState[VIEW_TODO.BASE].fetching to show loader or not.